.pre-commit-config.yaml

```diff
@@ -10,22 +10,24 @@ repos:
   hooks:
   - id: isort
 - repo: https://github.com/psf/black
-  rev: 23.3.0
+  rev: 23.7.0
   hooks:
   - id: black
-    language_version: python3
+    args: [--target-version=py37]
```

Comment:
please add the target-version config via pyproject.toml

```toml
[tool.black]
target-version = ['py37']
```

Reply:
Done